app/src/debug/res/xml/shortcuts.xml

```diff
@@ -1,34 +1,34 @@
 <?xml version="1.0" encoding="utf-8"?>
-<shortcuts xmlns:tools="http://schemas.android.com/tools"
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     tools:targetApi="n_mr1">
 
     <shortcut
-        android:shortcutId="search"
         android:icon="@drawable/shortcut_search"
+        android:shortcutId="search"
```

Comment:
I think it makes sense for `id` go first as it arguably has a higher priority than the other attributes. Same for other shortcuts.